config/initializers/feature_toggle.rb

```diff
@@ -20,4 +20,10 @@ def self.location_toggle_enabled?
 
     false
   end
+
+  def self.managing_for_other_user_enabled?
+    return true unless Rails.env.production?
+
+    false
+  end
```

Comment:
I've updated the rest of these toggles, hope you agree this is simpler:

```suggestion
  def self.managing_for_other_user_enabled?
    !Rails.env.production?
  end
```